autogpt_platform/frontend/src/components/contextual/GoogleDrivePicker/useGoogleDrivePicker.ts

```diff
@@ -242,14 +251,27 @@ export function useGoogleDrivePicker(options: Props) {
   }
 
   function buildAndShowPicker(accessToken: string): void {
+    if (!developerKey) {
+      const error = new Error(
+        "Google Drive Picker requires NEXT_PUBLIC_GOOGLE_API_KEY to be set",
+      );
+      console.error("[useGoogleDrivePicker]", error.message);
+      if (onError) onError(error);
```

Comment:
I think `onError` is not optional on the props so no need for the `if` here? Also the error above doesn't look too user friendly, maybe something more like `Mising Google Drive configuration`?